lib/sentry/envelope.ex

```diff
@@ -34,6 +34,17 @@ defmodule Sentry.Envelope do
     }
   end
 
+  @doc """
+  Creates a new envelope containing the client report and a map of discarded events.
+  """
+  @spec from_client_report(ClientReport.t()) :: t()
```

Comment:
This module is private but since we have a doc maybe this helps:

```suggestion
  @doc since: "10.8.0"
  @spec from_client_report(ClientReport.t()) :: t()
```